documentation/modules/ROOT/pages/index.adoc

```diff
@@ -77,3 +77,17 @@ their Python applications.
 [.link]
 https://openmanufacturingplatform.github.io/sds-documentation/python-sdk-guide/index.html[Python SDK]
 --
+
+[.text]
```

Comment:
Every tile must have the following structure:
```
[.tile]
[.icon-cli]
--
[.title]
THE TITLE
[.text]
THE SUMMARY TEXT
[.link]
THE URL OR XREF
--
```

Where `.icon-cli` is the CSS class to use for this tile, this is currently unused in the [esmf-antora-ui](https://github.com/eclipse-esmf/esmf-antora-ui), but should be present.